src/website/app/demos/Dialog/examples/DialogTitle/basic.js

```diff
@@ -0,0 +1,34 @@
+/* @flow */
+import { DialogTitle, DialogHeader } from '../../../../../../library/Dialog';
+import DemoLayout from '../../components/DemoLayout';
+import Dialog from '../../components/DemoDialog';
+
+export default {
+  id: 'basic',
+  title: 'Basic Usage',
+  description: `Display a title with predefined Mineral styles. Providing
+DialogTitle text also automatically create an accessible label for the Dialog.
```

Comment:
@kylegach I think we discussed that no matter what you provide to DialogTitle, it will get an accessible label via aria-labelledby, correct? If so, this description should  be more like "Display a title with predefined Mineral styles. Implementing a DialogTitle automatically creates an accessible label for the Dialog." (Note that either way, "create" should be "creates" - probably my bad!)